suite/runsuite_wrapper.pl

```diff
@@ -313,6 +313,11 @@
                                    'code_api|linux.fib-conflict' => 1,
                                    'code_api|linux.fib-conflict-early' => 1,
                                    'code_api|linux.mangle_asynch' => 1,
+                                   'code_api|tool.drcachesim.phys' => 1, # i#4922
+                                   'code_api|tool.drcacheoff.rseq' => 1, # i#4924
```

Comment:
This can be removed since the test mode fix in this PR fixes it?

Reply:
This test would then be removed from the commit message too?

Reply:
> This can be removed since the test mode fix in this PR fixes it?

The test now runs with the `test_mode_flag` fix but the test fails with `failed to match expected output`. Updated #4924.